src/common/desktop/DesktopUtils.ts

```diff
@@ -260,6 +254,36 @@ export class DesktopUtils {
 			return w.commonNativeFacade.createMailEditor(/* filesUris */ [], /* text */ "", /* addresses */ [], /* subject */ "", /* mailtoURL */ mailToArg)
 		}
 	}
+
+	relaunch() {
+		const appImage = this.process.env.APPIMAGE
+		if (this.electron.app.isPackaged && appImage) {
```

Comment:
like you said, would be great to check the signature, but until then can we make sure we only use `$APPIMAGE` on Linux? My concern is malware on Windows using the app as a trampoline but on Linux it is less off a concern. And, well, this reduces the attack surface either way.